LeaderboardBackend/Models/ViewModels/JudgementViewModel.cs

```diff
@@ -17,7 +17,7 @@ public record JudgementViewModel
 	///   When the judgement was made. Follows <a href="https://www.ietf.org/rfc/rfc3339.txt">RFC 3339</a>.
 	/// </summary>
 	/// <example>2022-01-01T12:34:56Z / 2022-01-01T12:34:56+01:00</example>
-	public string CreatedAt { get; set; }
+	public string? CreatedAt { get; set; }
```

Comment:
A little iffy about this one since `CreatedAt` shouldn't ever be `null`

Reply:
It doesn't really matter, cause this is an output model. We don't do any data validation on it.

Reply:
Mmmmmmmmmm fine